content/docs/concepts/oracle-server/_index.md

```diff
@@ -0,0 +1,37 @@
+
+---
+title: "Autonity Oracle Server (AOS)"
+linkTitle: "Oracle Server"
+weight: 4
+description: >
+  Oracle server software features
+---
+
+AOS is the reference implementation of the Autonity Oracle Protocol and the oracle server software run by validator nodes in an Autonity network. It is composed of off- and on-chain components.
+
+## Features
+
+### Core logic
+
+- _Core_, the core off-chain Autonity Oracle Server codebase managing interactions with the external price data source providers via a data adaptor 'plugin' architecture, and the on-chain Oracle Contract via the connected Autonity Go Client node. Core executes aggregation of data from external sources ("_L1 aggregation_"), calculates a median price, and submits price report transactions on-chain to the Oracle Contract.
```

Comment:
should we use abbreviations more frequently in this section?  AOS, AGC, AOC, etc, and IMHO language is a bit more abstract. It is just an opinion though. for e.g suggested change in networking para:
> For networking, the system uses WebSocket and HTTP connections. HTTP RPC calls are made to configured data source providers, and a WebSocket connection is established with the AGC validator to (a) submit price report transactions and (b) listen for on-chain Oracle Contract events.

Reply:
Yes, you're right. I've edited _networking_ and the other entries in the _Core logic_ entry to simplify per your suggestion.  Commit [dd108fb](https://github.com/autonity/docs.autonity.org/pull/75/commits/dd108fb92d91f6ca01d4f2661fb3892410ad1f2a).

Reply:
@vpiyush does this edit address your comment, please?